### ISSUE TITLE
Counter for refs, interactions, and unique interactors not working 

### ISSUE THREAD

**Issue by jsimons9** [GoMexSI/gomexsi-web#110]
I have noticed for a while now that the number of references, interactions, and unique interactors has not changed on the web page even though we have uploaded a large number of new extracted references.  Not sure who would be the one to fix this issue.

**Comment by jhpoelen** [comment]
Interesting! Your general description describe a high level issue. Can you provide some more details? For instance: What are the observed numbers? What are the expected numbers? When did the upload occur?

**Comment by jsimons9** [comment]
The current numbers are 197 references, 2,572 unique interactors, and 103,861 total interactions.  That number has not changed in some time, and all of the uploads done by Jordan were done recently and those numbers did not change. I am not really sure what the expected numbers are as I do not know from how far back the current numbers go.  Jordan's uploads occurred in early September, almost a month and a half ago.

**Comment by jhpoelen** [comment]
Ok, thanks for detailing the current numbers. So, if I am correct, you are saying that the number did not change since Jordan's updates, and you are not quite sure what the numbers were before that. 

As far as I know, Jordan made her work available via https://github.com/jlew3315/JLewis_GoMexSi . At the time, I took some time to review the work and made some suggestions https://github.com/jlew3315/JLewis_GoMexSi/pull/1 . These suggestion are still waiting to be accepted by Jordan. 

Also, the dataset with corrections is available via https://github.com/jhpoelen/JLewis_GoMexSi . This dataset has been included in GloBI, contained 342 predator lines, 1842 prey lines, and 33 references. The resulting records are available through GloBI and, by proxy, via https://gomexsi.tamucc.edu . I've attached a screenshot of a search for records of Bottlenose dolphin (_Tursiops truncatus_) as an example.

Please provide specific examples of records that Jordan provided, but are not yet available through https://gomexsi.tamucc.edu, because at this point,  I cannot find any evidence suggesting that Jordan's work did not make it to GoMexSI. 

![screenshot from 2018-10-26 09-24-20](https://user-images.githubusercontent.com/1084872/47579655-7bcc9580-d901-11e8-8ee9-e59622959e5a.png)

**Comment by jsimons9** [comment]
Hi Jorrit,  
Finally getting some time to respond.  I guess I am not doubting that the data are in GoMexSI, but rather the counter that appears on the homepage of the webpage is not working correctly.  I am attaching a screenshot of the webpage from January of 2018, well before Jordan uploaded her data, and the image indicates that there are 197 references, and that is the same number I see right now as I look at the webpage just called up.  If she uploaded 33 references then that number should be at least 230.  I assume that Reed set up the counter but I am not sure.  Can you comment on the possible location of the problem.  Thanks so much.
[GoMexSI webpage counter Jan 2018.pdf](https://github.com/GoMexSI/gomexsi-web/files/2658539/GoMexSI.webpage.counter.Jan.2018.pdf)

PS.  I will be sending the slides from my talk in Mexico in a moment.

**Comment by jhpoelen** [comment]
Thanks for sharing. Can you provide some examples of references and associated interactions that you'd expect to show up but cannot find via GoMexSI / GloBI ?

**Comment by jsimons9** [comment]
No, it just the counter that is not working.  There is no way to associate the missing counts to any data.  It just tells us how many references, unique interactors and total interactions there are in GoMexSI. I am assuming the data are there, it is just the counter that is not working.  I am not sure I am explaining this well enough.

**Comment by jhpoelen** [comment]
I'd suggest to use the open tools at your disposal (e.g., rglobi, cypher, datasets at https://globalbioticinteractions.org/data , datasets in GitHub) to narrow down the issue. With these tools you can calculate your desired metrics and compare them to those presented on the GoMexSI homepage.
